docs/http-api/projections.md

```diff
@@ -209,6 +221,8 @@ The server then returns the state for the partition:
 * `emit`: Allow the projection to write to streams (true/false)
 * `trackemittedstreams`: Write the name of the streams the projection is managing to a separate stream. $projections-{projection-name}-emittedstreams (true/false)
 
+### Delete a Projection
+
 | URI                                                                                                                                                    | Description                                                                                     | HTTP verb |
 |:-------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------|:----------|
 | `/projection/{name}`                                                                                                                                   | Returns information for a projection.                                                           | GET       |
```

Comment:
Shouldn't the verb be delete?


```suggestion
| `/projection/{name}`                                                                                                                                   | Returns information for a projection.                                                           | DELETE       |
```

Reply:
Ah there were actually two different actions in the table. I think the original formatting was meant to be closer to what's in the most recent commit. It's up to you whether that's more readable than different sections per action @cdevarenne @rsowald 🙂